docs/docs/getting-started/prerequisites.mdx

```diff
@@ -29,13 +29,13 @@ On Ubuntu you may need to run:
 :::caution Alpine Linux is not yet supported
 Pants for Linux is currently distributed as a manylinux wheel. Alpine Linux is not covered by manylinux (it uses MUSL libc while manylinux requires glibc), so at present Pants will not run on Alpine Linux.
 
-If you need to run Pants on Alpine, [let us know](/community/members), so we can prioritize this work. Meanwhile, you can try [building Pants yourself](./manual-installation.mdx#building-pants-from-sources) on Alpine.
+If you need to run Pants on Alpine, [let us know](/community/members), so we can prioritize this work. Meanwhile, you can try [building Pants yourself](./installing-pants.mdx#running-pants-from-sources) on Alpine.
 :::
 
-:::caution Linux on ARM will be supported from Pants 2.16
-Pants 2.16 will be distributed for Linux x86_64 and ARM64. Earlier versions are only distributed for Linux x86_64.
+:::caution Linux on ARM is supported from Pants 2.16
+Pants 2.16 and newer is distributed for Linux x86_64 and ARM64. Earlier versions are only distributed for Linux x86_64.
```

Comment:
(This is an unrelated minor change that I noticed in passing. We're well past 2.16 and ARM Linux is well supported now.)